R/model_parameters.psych.R

```diff
@@ -41,16 +42,14 @@
 #' }
 #'
 #' # FactoMineR ---------
-#' \dontrun{
```

Comment:
Are examples not run when they're wrapped in `donttest{}`? Else, this should be reverted.

Reply:
The examples are not run when wrapped in `donttest{}`, so this shouldn't change the behavior.